SCHEMA-SPEC.md

```diff
@@ -258,11 +258,11 @@ and property names when the `node-names` or `prop-names` options are activated.
     * `idn-hostname`: RFC5890 internationalized internet hostname.
     * `ipv4`: RFC2673 dotted-quad IPv4 address.
     * `ipv6`: RFC2373 IPv6 address.
-    * `uri`: RFC3986 URI.
-    * `uri-reference`: RFC3986 URI Reference.
-    * `iri`: RFC3987 Internationalized Resource Identifier.
-    * `iri-reference`: RFC3987 Internationalized Resource Identifier Reference.
-    * `uri-template`: RFC6570 URI Template.
+    * `url`: RFC3986 URI.
```

Comment:
Should this still say "RFC3986 URI"?

Reply:
yep! It's still a URI. We're just using `url` as a label for ergonomics :)

Reply:
Okay, just checking!